### PR TITLE
improve run riscv isa test suite.sh

### PR DESCRIPTION
**Pull request by StanSofomoDev**
PR for https://sofomo.atlassian.net/browse/CN-81 and https://sofomo.atlassian.net/browse/CN-80

### CODE REVIEW

File: tests/isa_test_suite/run_riscv_isa_test_suite.sh [hunk 1]
```diff
@@ -26,17 +27,40 @@ fi
...
+        2)
+            echo "Stopping all tests..."
+            stop_all_tests=true
+            ;;
+        *)
+            echo "Invalid choice. Continuing..."
+            ;;
+    esac
+}
+
```
Comment (jeffnye-gh) [inline tests/isa_test_suite/run_riscv_isa_test_suite.sh:54]: why do i need this? just exit everything.

File: README.md [hunk 2]
```diff
@@ -233,12 +233,12 @@ Logs from running the `regress` target are available for review. Logs are locate
...
 
-  To update `disabled_isa_tests.txt` run:
+  To update `isa_tests_list.txt` run:
 
   ```bash
   cd tests/isa_test_suite
   make
-  bash update_disabled_isa_tests_list.sh
+  bash update_isa_tests_list.sh
   ```
```
Comment (jeffnye-gh) [inline README.md:242]: this is only for people that update the sha for riscv tests right?

File: README.md [hunk 3]
```diff
@@ -223,7 +223,7 @@ Logs from running the `regress` target are available for review. Logs are locate
 
 - **stf_gen_ad_hoc:** This test generates and compares traces for various bare metal operations. It extracts ELF files, runs load/store operations, and compares the generated traces against golden (reference) traces to ensure they match. This process is done for both uncompressed and compressed traces, with the aim of validating the integrity of the STF (System Trace Format) files in different configurations.
 
-- **riscv_isa_test:** This test runs a suite of RISC-V ISA compliance tests using a predefined list of enabled tests. It simulates each test and compares the results to verify correct functionality. Tests can be enabled or disabled by moving files between the respective `enabled_isa_tests.txt` and `disabled_isa_tests.txt` test lists in the isa_test_suite directory. The `disabled_isa_tests.txt` file is only for documentation purposes and does not affect how the test runs. 
+- **riscv_isa_test:** This test runs a suite of RISC-V ISA compliance tests using a predefined list of enabled tests. It simulates each test and evaluates the results to verify correct functionality. Tests can be enabled/disabled in `isa_tests_list.txt` in the isa_test_suite directory.
```
Comment (jeffnye-gh) [inline README.md:226]: and where do you tell me how to enable/disable tests?

File: README.md (hunk 2 above)
Comment (jeffnye-gh) [reply README.md:242]: how does an end user know what this is for?

**Review by jeffnye-gh — CHANGES_REQUESTED** [review]

File: tests/isa_test_suite/run_riscv_isa_test_suite.sh (hunk 1 above)
Comment (StanSofomoDev) [reply tests/isa_test_suite/run_riscv_isa_test_suite.sh:36]: Changed

File: README.md (hunk 2 above)
Comment (StanSofomoDev) [reply README.md:242]: I've added information about it.

File: README.md (hunk 3 above)
Comment (StanSofomoDev) [reply README.md]: Updated

**Review by jeffnye-gh — APPROVED** [review]

**Comment by jeffnye-gh** [comment]
DO NOT MERGE THIS BRANCH.

I have merged this into a local branch which will be merged to main.

**Pull request merged**